thirdfiledemo.txt

```diff
@@ -3,4 +3,9 @@
 "third file for demo fifth line- Line 5" 
 "third file for demo" 
 "third file for demo fourth line- Line 4" 
+"somee new changes" 
```

Comment:
This is fine

Reply:
resolved the comment/ incorporated the changes